presentation.md

```diff
@@ -86,14 +86,13 @@ permalink: /presentation/
   utbildningsväsen. Styrelsen har följande ledamöter:
 
   - Länsrådet Lennart Johansson, ordförande
-  - Professor Magnus Hellgren, vice ordförande
+  - Verkställande direktör Magnus Hellgren, vice ordförande
   - Professor Peter Aronsson
   - Förutvarande kommundirektör Irene Hedfors
-  - Verkställande direktör Magnus Hellgren
   - Landstingsledamot Christina Lindqvist
   - Utbildningskonsult Gudrun Lundqvist
-  - Sekreterare är Björn Hedin.
-
+  - Professor Börje Nilsson
+  
```

Comment:
Did you intend to add two spaces here? Two spaces is typically used for adding linebreaks at end of lines, but I think it serves a purpose here?